packages/lodestar/src/executionEngine/http.ts

```diff
@@ -113,6 +126,9 @@ export class ExecutionEngineHttp implements IExecutionEngine {
         {
           retryAttempts: this.retryAttempts,
           retryDelay: this.retryDelay,
+          onEachRetryFn: () => {
+            this?.metrics?.executionEngineRequestCount.inc({method});
```

Comment:
this can be undefined?

Reply:
I am removing this because i am moving metrics inside json rpc